### ISSUE TITLE
Cannot disable controller `restic-repo`

### ISSUE THREAD

**Issue by Lirt** [vmware-tanzu/velero#5605]
Hello Velero community,

**What steps did you take and what happened:**

I tried to run with disabled restic controller - `velero server --disable-controllers=restic-repo`. The outcome is that `restic-repo` is not supported as disabled controller while `--help` says it is one of valid options.

```bash
2022/11/19 10:53:58 Invalid value for --disable-controllers flag provided: restic-repo. Valid values are: backup,backup-deletion,backup-sync,download-request,gc,restic-repo,restore,schedule,server-status-requestunable to disable a controller
```

**What did you expect to happen:**

That it will run.

**The following information will help us better understand what's going on**:

**Anything else you would like to add:**

- If this option is not supported, then this is a documentation issue. If confirmed I can create a PR to update documentation to properly reflect this.
- If this option should be supported I can take a look into code and try to make a PR to make it work.

**Environment:**

- Velero version (use `velero version`): 
```yaml
Client:
    Version: v1.9.3
    Git commit: 9a617fe5af9c997d69f8f37a5b40d0462edd9b28
Server:
    Version: v1.9.3
```
- Velero features (use `velero client config get features`): `features: <NOT SET>`
- Kubernetes version (use `kubectl version`): 
```bash
Client Version: version.Info{Major:"1", Minor:"25", GitVersion:"v1.25.2", GitCommit:"5835544ca568b757a8ecae5c153f317e5736700e", GitTreeState:"clean", BuildDate:"2022-09-21T14:33:49Z", GoVersion:"go1.19.1", Compiler:"gc", Platform:"linux/amd64"}
Kustomize Version: v4.5.7
Server Version: version.Info{Major:"1", Minor:"24", GitVersion:"v1.24.7", GitCommit:"e6f35974b08862a23e7f4aad8e5d7f7f2de26c15", GitTreeState:"clean", BuildDate:"2022-10-12T10:50:21Z", GoVersion:"go1.18.7", Compiler:"gc", Platform:"linux/amd64"}
```
- Kubernetes installer & version: 1.24.7 - CAPI
- Cloud provider or hardware configuration: Openstack

**Comment by allenxu404** [comment]
Thanks for pointing it out. Yes, restic controller is expected to be on the list in which Velero allows users to disable those controller and it doesn't take effect.
We've already fixed this bug on v1.10.

**Comment by Lirt** [comment]
Thank you for answer. I will wait for 1.10.
